terraform/modules/data-share-service/variables.tf

```diff
@@ -4,7 +4,16 @@ variable "ecr_url" {}
 variable "cloudwatch_retention_period" {}
 variable "vpc_cidr" {}
 variable "lev_url" {}
+
 variable "db_username" {}
 variable "externally_allowed_cidrs" {
   type = list(string)
 }
+
+variable "prisoner-event-enabled" {}
+variable "prisoner-search-url" {}
+variable "hmpps-auth_url" {}
+variable "prisoner-search-client-id" {}
+variable "prisoner-search-client-secret" {}
+variable "prisoner-event-queue-name" {}
+variable "prisoner-event-dlq-name" {}
```

Comment:
Could we change these from kebab to snake case please :)

Reply:
done!